src/core/ext/client_config/client_channel.c

```diff
@@ -33,6 +33,7 @@
 
 #include "src/core/ext/client_config/client_channel.h"
```

Comment:
For late night debugging sanity:

Can we add some 3+ line comments on where the call_data and where the channel_data parts of the file are?

I'm thinking:

```c
/*************************************************************************
 * CHANNEL-WIDE FUNCTIONS
 */

/*************************************************************************
 * PER-CALL FUNCTIONS
 */
```